<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-Q8UQAUDS" ignored="1" name="Default test(failed)(failed)" tests="5" failures="4" timestamp="2022-06-27T08:53:18 IST" time="0.194" errors="0">
  <testcase name="test4" time="0.029" classname="testNG2.Sample2">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNG2.Sample2.test4(Sample2.java:35)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test4 -->
  <testcase name="test3" time="0.003" classname="testNG2.Sample2">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNG2.Sample2.test3(Sample2.java:28)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test5" time="0.002" classname="testNG2.Sample2">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNG2.Sample2.test5(Sample2.java:42)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test5 -->
  <testcase name="test2" time="0.002" classname="testNG2.Sample2">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNG2.Sample2.test2(Sample2.java:21)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test1" time="0.002" classname="testNG2.Sample2"/>
  <testcase name="test6" time="0.0" classname="testNG2.Sample2">
    <ignored/>
  </testcase> <!-- test6 -->
</testsuite> <!-- Default test(failed)(failed) -->
